_posts/2018-11-11-kanban.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "My Kanban Journey"
+category: puppet
+tags: puppet agile kanban
+---
+
+***Disclaimer: This post was written as an employee of Puppet, Inc., but does not constitute opinions of the company.***
+
+Last year I was part of the effort to get the [Puppet Development Kit](https://puppet.com/blog/develop-modules-faster-new-puppet-development-kit) built and shipped. Since I'm very interested in team effectiveness and agility, at one point I took a day to re-organize our kanban boards to streamline the info exposed and reduce the friction when using them. Only after leaving the team, I realised [the value of having a low-friction JIRA configuration](https://twitter.com/nigelkersten/status/1024675669429518336):
+
+![a screenshot of the tweet linked above](/log/assets/2018-11-11-kanban/tweet.png)
+
+Thanks due to some prodding by [@sigje](https://twitter.com/sigje/status/1025121668191023104) and some internal folks, I've finally come around to writing this up.
+
+![a screenshot of the tweet linked above](/log/assets/2018-11-11-kanban/prod.png)
+
+I'll start out with an overview of how the boards look like today, and will have the life of a ticket in the second half.
```

Comment:
You mention starting out with an overview of the boards, but then jump into Dual Track agile.  Consider rewording to talk about  how dual track agile inspired/influence/affected how you wanted to view work, the overview of the boards and the life cycle of a ticket in the new world order!

Reply:
Updated to reflect the structure of the post.